pheno005/phenotype/demographics.json

```diff
@@ -0,0 +1,75 @@
+{
+	"participant_id": {
+		"Description": "OpenNeuro ID of the subject."
+	},
+	"session_id": {
+		"Description": "ID assigned to participant's visit.",
+		"Levels": {
+			"baseline": "First imaging and phenotype data collection visit.",
+			"followup": "Second phenotype data collection visit."
+		}
+	},
+	"run_id": {
+		"Description": "ID assigned to a run for a given session.",
+		"Levels": {
+			"01": "First run of the session.",
+			"02": "Second run of the session."
```

Comment:
These should be `run-01` and `run-02`, respectively. The same should be in the TSV.